chore: Fix Attachments and Supplement storage definitions

TheTechmage opened the pull request:
This should fix the issue that @dbluhm mentioned

TheTechmage committed:
chore: Fix Attachments and Supplement storage definitions

Signed-off-by: Colton Wolkins (Indicio work address) <colton@indicio.tech>

dbluhm commented on aries_cloudagent/protocols/present_proof/v2_0/models/pres_exchange.py:
```diff
@@ -334,6 +334,6 @@ class Meta:
     supplements = fields.Nested(
```
Need to set `many` to `True`

dbluhm commented on aries_cloudagent/protocols/present_proof/v2_0/models/pres_exchange.py:
```diff
@@ -334,6 +334,6 @@ class Meta:
         SupplementSchema(), required=False, description="Supplementary attachment data"
     )
-    attach = fields.Nested(
+    attachments = fields.Nested(
```
Same

TheTechmage committed:
fix: Switch schema definitions to many=True

Both Supplement & Attachments were not explicitely allowing many items.
Now they do.

Signed-off-by: Colton Wolkins (Indicio work address) <colton@indicio.tech>

TheTechmage commented:
The above changes from Peter were not meant to be a part of this PR. I will work to get those separated into another PR

TheTechmage commented:
Force push removed the previous two commits. I have moved those to a separate branch, feature/credential-attachments-verifier-tests

TheTechmage committed:
fix: attach -> attachment references

Signed-off-by: Colton Wolkins (Indicio work address) <colton@indicio.tech>

TheTechmage committed:
chore: Fix typo

Signed-off-by: Colton Wolkins (Indicio work address) <colton@indicio.tech>

dbluhm reviewed (approved):
Looks good, thanks!
